apps/state/lib/state/stops_on_route.ex

```diff
@@ -198,6 +198,12 @@ defmodule State.StopsOnRoute do
        ["Boat-Aquarium", "Boat-Fan", "Boat-Logan", "Boat-Quincy", "Boat-Winthrop"]}
     ]
 
+  defp gather_explicit_overrides(%{id: "Green-D"}, 0),
+    do: [
+      {"Green-D", 0, :all, "Green-SmrGLXNorthSta-Std", true,
```

Comment:
Yep, this will get added in the next push momentarily. As a note, I realized that these overrides have to specify each stop, so you'll also see that every single station on the line will now be noted (sigh)...